ckanext/bulk/zipoutput.py

```diff
@@ -91,7 +91,12 @@ def schema_to_csv(typ, schema_key, objects):
         header.append(field['label'].encode('utf8'))
     w.writerow(header)
     for obj in sorted(objects, key=lambda p: p['name']):
-        w.writerow([obj.get(field_name, '').encode('utf8') for field_name in field_names])
+        # fix for bpa-archive-ops#792 AttributeError: 'int' object has no attribute 'encode'
```

Comment:
no need to put comments in referencing the ticket, reference it from the commit if relevant